libsigner/src/tests/http.rs

```diff
@@ -0,0 +1,347 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
+// Copyright (C) 2020-2023 Stacks Open Internet Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use std::collections::HashMap;
+use std::io;
+use std::io::Read;
+use std::io::Write;
+use std::str;
+
+use crate::error::EventError;
+use crate::error::RPCError;
+use crate::http::{decode_http_body, decode_http_request, decode_http_response, run_http_request};
+
+use stacks_common::util::chunked_encoding::*;
+
+#[test]
```

Comment:
This can also probably all go away since we're using `tiny_http`

Reply:
Lines 29 through 101 can but until we update SignerSession, we can't get rid of the rest.